src/phyling/libphyling.py

```diff
@@ -104,23 +106,19 @@ def trim_gaps(
     """Trim the locations on MSA if the gappyness ratio (gaps/total length) is higher than a the arg gaps."""
     if gaps > 1 or gaps < 0:
         raise ValueError('The argument "gaps" should be a float between 0 to 1.')
-    infoList = [{"id": rec.id, "name": rec.name, "description": rec.description} for rec in pep_msa]
-    msa_array = np.array([list(rec) for rec in pep_msa])
-    gappyness = (msa_array == "-").mean(axis=0)
-    pep_trimList = np.where(gappyness > gaps)[0]
-    if pep_trimList.size > 0:
-        msa_array = np.delete(msa_array, pep_trimList, axis=1)
+    
+    clipkit_pep_msa = MSA.from_bio_msa(pep_msa)
+    
+    clipkit_pep_msa.trim(mode=TrimmingMode.gappy, gap_threshold=gaps)
 
     if cds_msa:
-        infoList = [{"id": rec.id, "name": rec.name, "description": rec.description} for rec in cds_msa]
-        msa_array = np.array([list(rec) for rec in cds_msa])
-        if pep_trimList.size > 0:
-            cds_trimList = np.concatenate([np.arange(num * 3, (num + 1) * 3) for num in pep_trimList])
-            msa_array = np.delete(msa_array, cds_trimList, axis=1)
-
-    return MultipleSeqAlignment(
-        [SeqRecord(Seq("".join(rec)), **info) for rec, info in zip(msa_array.tolist(), infoList)]
-    )
+        clipkit_cds_msa = MSA.from_bio_msa(cds_msa)
+        if clipkit_pep_msa.sites_trimmed.size > 0:
+            cds_site_positions_to_trim = np.concatenate([np.arange(num * 3, (num + 1) * 3) for num in clipkit_pep_msa.sites_trimmed])
```

Comment:
`clipkit_pep_msa.sites_trimmed` will return all the sequences on the sites being trimmed. E.g.
```
array([['G', '-', '-'],
       ['-', 'G', 'T']], dtype='<U1')
```
But we're not able to use this to do back-translation. We need locations instead of the sequences itself. I guess the clipkit_pep_msa._site_positions_to_trim would return the proper data we need. E.g.
```
array([1, 2, 3])
```